README.md

```diff
@@ -1,6 +1,5 @@
 # JS Stellar Base
 
-[![Build Status](https://travis-ci.com/stellar/js-stellar-base.svg?branch=master)](https://travis-ci.com/stellar/js-stellar-base)
```

Comment:
FYI you can create a new badge for GitHub Actions at this URL, you click the `...` in the top right:

https://github.com/stellar/js-stellar-base/actions/workflows/gulp.yml

Reply:
Cool! Done in https://github.com/stellar/js-stellar-base/pull/457/commits/a03ba59f35e894747f5f45d18188b7d8510fc6ef